dm/checker/check_test.go

```diff
@@ -67,7 +67,7 @@ func TestIgnoreAllCheckingItems(t *testing.T) {
 	require.NoError(t, err)
 	require.Len(t, msg, 0)
 
-	result, err := RunCheckOnConfigs(context.Background(), nil)
+	result, err := RunCheckOnConfigs(context.Background(), nil, false)
```

Comment:
Do we need a test where `dumpWholeInstance ` is true?

Reply:
https://github.com/pingcap/tiflow/pull/7331/files#diff-f908b4efd00a69aa510a8c86679d44f92bb01f08ec21c0a653f4abe0c6b4127bR182-R191